vmaas_sync/metrics.go

```diff
@@ -86,11 +87,24 @@ var (
 		Subsystem: "vmaas_sync",
 		Name:      "backend_processes",
 	}, []string{"state"})
+
+	cyndiStats = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "vmaas_sync",
+		Name:      "cyndi_data",
+	}, []string{"type"})
+
+	cyndiCnt = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "vmaas_sync",
+		Name:      "cyndi_systems",
+	}, []string{"last_upload"})
```

Comment:
We can use only one lable `type` here. We can use it for "last_upload" flags (last1D, last7D...) and also for "tags_used" flag (count of systems with at least one tag), "sap_system" (count of systems with sap_system=true).